proof-systems/src/darlin/poly-commit/src/constraints/tests.rs

```diff
@@ -546,11 +672,43 @@ pub(crate) fn single_point_multi_poly_test<
             max_num_queries: 1,
             segmented: false,
             negative_type,
+            marlin_params: None,
         })
         .unwrap()
     })
 }
 
+pub(crate) fn succinct_verify_with_marlin_params_test<
+    ConstraintF: PrimeField,
+    G: Group<BaseField = ConstraintF>,
+    PC: 'static + PolynomialCommitment<G, Commitment = G>,
+    PCG: 'static + PolynomialCommitmentVerifierGadget<ConstraintF, G, PC>,
+>() {
+    test_multi_point_multi_poly_verify::<
+        ConstraintF,
+        G,
+        DomainExtendedPolynomialCommitment<G, PC>,
+        DomainExtendedPolyCommitVerifierGadget<ConstraintF, G, PC, PCG>,
+    >(
+        TestInfo{
+            num_iters: 1,
+            max_degree: Some(1usize << 19),
+            supported_degree: Some(1usize << 17),
+            num_polynomials: 0,
+            max_num_queries: 0,
+            segmented: false,
+            negative_type: None,
```

Comment:
You can replace this with `...` so you are not forced to specify a value for them if you are not interested

Reply:
Ok, thanks